### PR TITLE
Create ko.json

### PR DESCRIPTION
**Pull request by totoromaum**
Hi, this is a translation into korean. Thank you.

### CODE REVIEW

File: language/ko.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        },
+        {
+          "label": "노트가 닫혔을 때",
+          "default": "보기가 연습 문제로 전환되었습니다."
+        }
+      ]
+    }
+  ]
+}
+© 2021 GitHub, Inc.
```
Comment (otacke) [inline language/ko.json:86]: Please remove those lines. Feel free to use https://translate-h5p.tk/ to translate H5P content types. It's not prone to those errors and offers other benefits as well.

**Review by otacke — CHANGES_REQUESTED** [review]
Thanks! Please be careful when working directly on github (instead of using git and an editor that helps you prevent mistakes or https://translate-h5p.tk/) . There are lines that should not be there. Those need to be deleted before I can merge this in.

File: language/ko.json (hunk 1 above)
Comment (totoromaum) [reply language/ko.json]: Hi, @otacke. I find this translation site, https://translate-h5p.tk/, is inconvenient for me. I will make sure that I submit immaculate code. Thank you. I updated the code above.

Comment (otacke) [reply language/ko.json]: Whatever ia better for you. Have checked your latest changes with a JSON checker?

Comment (totoromaum) [reply language/ko.json]: Hi, @otacke, I updated the code and checked the updated version with a JSON checker. I hope this version is all right. Thank you.

**Review by otacke — APPROVED** [review]
This one is looking outstanding. Thanks!

**Pull request merged**